package.json

```diff
@@ -6,6 +6,10 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "cors": "^2.8.5",
+    "express": "^4.17.1",
+    "mongoose": "^5.9.23",
+    "mongose": "0.0.2-security",
```

Comment:
Please remove line 12. I think it was a typo when you wanted to install it first time. since the right one is there, line 12 should be removed